src/modules/Dashboard/features/RespondentData/RespondentDataSummary/hooks/useRespondentAnswers/useRespondentAnswers.utils.ts

```diff
@@ -12,23 +17,84 @@ export const getDateISO = (date: Date, time: string) => {
   return new Date(utcDate).toISOString().split('.')[0];
 };
 
-export const getIdentifiers = (
-  filterByIdentifier = false,
-  filterIdentifiers = [] as AutocompleteOption[],
-  identifiers = [] as Identifier[],
-): string[] | undefined => {
-  if (!filterByIdentifier) return;
+export const getIdentifiers = ({
+  filterByIdentifier,
+  filterIdentifiers = [],
+  identifiers = [],
+}: GetIdentifiers): GetIdentifiersReturn => {
+  if (!filterByIdentifier) return null;
 
   const filterIds = new Set(filterIdentifiers.map((identifier) => identifier.id));
 
-  return identifiers.reduce(
-    (decryptedIdentifiers: string[], { encryptedValue, decryptedValue }) => {
+  const { selectedIdentifiers, answerDates } = identifiers.reduce(
+    (
+      acc: { selectedIdentifiers: string[]; answerDates: string[] },
+      { encryptedValue, decryptedValue, lastAnswerDate },
+    ) => {
       if (filterIds.has(decryptedValue)) {
-        decryptedIdentifiers.push(encryptedValue);
+        acc.selectedIdentifiers.push(encryptedValue);
+        acc.answerDates.push(lastAnswerDate);
       }
 
-      return decryptedIdentifiers;
+      return acc;
     },
-    [],
+    { selectedIdentifiers: [], answerDates: [] },
   );
+
+  const recentAnswerDateString = answerDates.reduce(
```

Comment:
Let's use `Math.max` if the goal to find maximum value from array:
```suggestion
  const recentAnswerDateString = Math.max(...answerDates);
```

Reply:
It seems not working with Math.max(), because answerDates array includes date strings.
![image](https://github.com/ChildMindInstitute/mindlogger-admin/assets/116339462/89b2bd62-68d4-4e6d-82e5-d4f1a8f14520)